pandas/io/json/_normalize.py

```diff
@@ -226,14 +226,26 @@ def _json_normalize(
     Returns normalized data with columns prefixed with the given string.
     """
 
-    def _pull_field(js: Dict[str, Any], spec: Union[List, str]) -> Iterable:
+    def _pull_field_meta(js: Dict[str, Any], spec: Union[List, str]) -> Any:
```

Comment:
```suggestion
    def _pull_field(js: Dict[str, Any], spec: Union[List, str]) -> Any:
```

Reply:
Does this only return a scalar?

Reply:
yeah, changed for `js` as well, since the result pulled out is the value of `js` which is currently defined with `Any`. thanks!

Reply:
emm, i changed to `Scalar`, but some annotation errors raised in `internal/manager.py`, and even if i changed back to `Any`, still see these errors:
```
pandas/core/internals/managers.py:1109: error: Unsupported target for indexed assignment
pandas/core/internals/managers.py:1111: error: Unsupported target for indexed assignment
pandas/core/internals/managers.py:1145: error: Unsupported target for indexed assignment
pandas/core/internals/managers.py:1146: error: Unsupported target for indexed assignment
pandas/core/internals/managers.py:1160: error: Unsupported target for indexed assignment
pandas/core/internals/managers.py:1161: error: Unsupported target for indexed assignment
```

do you have any idea how/why those errors are raised? any hints are very appreciated, thanks! @WillAyd